app/views/jobs/_job_details.html.erb

```diff
@@ -1,20 +1,54 @@
-<%= label :job, :resident_name, 'Resdient Name', :class => 'col-form-label'  %>
-<%= job[:FirstName] %>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/divbs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+  
+<form>
+  <div class="form-row">
+    <div class="form-group col-md-4">
+    <%= label :job, :resident_name, 'Resident Name'  %>
+    <input type="text" class="form-control" id="resident_name" value="<%= job[:FirstName] %>" readonly>
```

Comment:
if you use `text_field` instead of `<input>`, it will be easy and 1-liner code to use `if` condition for `readonly` for edit view.